packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1610,6 +1610,23 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         };
     }
 
+    /**
+     * @param tree a summary tree
+     * @returns count of datastores and dds's in the tree
+     */
+    private countDataStore(tree): number {
+        let count = 0;
```

Comment:
The issue talks about collecting number of data stores / DDSs, i.e. number of trees at first level (as most important part). DDSs are interesting, but secondary.

Total number of all sub-trees is interesting, but it's Pri3.

It's also great to count how many of those data stores are not referrenced (i.e. have unreferenced === true flag).

Reply:
should I only log the count of unreferenced data stores?

Reply:
It would be great to get count of all data stores, and unreferenced data stores.
I assume here that it includes only data stores that produced summary (i.e. were not reused).
Pri2 would be to also log number of data stores that were not summarized, i.e. were reused from previous summary.